metadata.rb

```diff
@@ -6,9 +6,9 @@
 issues_url       'https://github.com/osuosl-cookbooks/dobc/issues'
 source_url       'https://github.com/osuosl-cookbooks/dobc'
 description      'Installs/Configures dobc'
-long_description 'Installs/Configures dobc'
 version          '2.0.4'
 
+depends          'base'
```

Comment:
What is base needed for?

Reply:
Was getting `undefined method 'osl_shell_environment' for cookbook: osl-docker` without base defined in Berksfile. I think it was getting it off the supermarket.